Tests/FunctionalTests/Sources/FTRSwiftTests.swift

```diff
@@ -69,6 +69,26 @@ class FunctionalTestRigSwiftTests: XCTestCase {
         .assertWithMatcher(grey_nil())
   }
 
+  func testInteractionWithALabelWithParentHidden() {
+    let checkHiddenBlock:GREYActionBlock =
+        GREYActionBlock.actionWithName("checkHiddenBlock", performBlock: { element, errorOrNil in
+                                        // Check if the found element is hidden or not.
+                                        let superView:UIView! = element as! UIView
```

Comment:
[CONTRIBUTING.md](https://github.com/google/EarlGrey/blob/master/CONTRIBUTING.md) lists `raywenderlich/swift-style-guide`. If that's not what we're doing, then it should be updated.

/cc @khandpur